### PR TITLE
bitCount documentation

### PR DESCRIPTION
**Pull request by otrazhenia**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/query_language/functions/bit_functions.md [hunk 1]
```diff
@@ -198,5 +198,36 @@ Result:
 │                    0 │
 └──────────────────────┘
 ```
+## bitCount {#bitcount}
 
+Calculates the number of bits set in the [integer](../../data_types/int_uint.md) number.
```
Comment (BayoNet) [inline docs/en/query_language/functions/bit_functions.md]:
```suggestion
Calculates the number of bits set to one in the [integer](../../data_types/int_uint.md) number.
```

Comment (BayoNet) [reply docs/en/query_language/functions/bit_functions.md]: Also, this function should support floating point types.

File: docs/en/query_language/functions/bit_functions.md [hunk 2]
```diff
@@ -198,5 +198,36 @@ Result:
...
+- Number of bits set.
+
+Type: [Integer](../../data_types/int_uint.md).
+
+**Example**
+
+Query:
+
+```
+SELECT bitCount(10)
```
Comment (BayoNet) [inline docs/en/query_language/functions/bit_functions.md]: But the result is for `SELECT bitCount(100)`

File: docs/en/query_language/functions/bit_functions.md [hunk 3]
```diff
@@ -198,5 +198,36 @@ Result:
...
+
+**Syntax** 
+
+```sql
+bitCount(x)
+```
+
+**Returned value**
+
+- Number of bits set.
```
Comment (BayoNet) [inline docs/en/query_language/functions/bit_functions.md]:
```suggestion
- Number of bits set to one in the input number.
```

File: docs/en/query_language/functions/bit_functions.md (hunk 1 above)
Comment (otrazhenia) [reply docs/en/query_language/functions/bit_functions.md]: You're right, I added it in the new version.

**Comment by BayoNet** [comment]
1. Link to the topic about floating-point types doesn't work.
2. You have lost the **Parameters** section in the description. Description of input values and their type should be in that section. So there shouldn't be this information in the first sentence of the description.

After doing this, translate to Russian.

**Pull request merged**